web/src/err.rs

```diff
@@ -0,0 +1,34 @@
+use std::fmt::Display;
+
+use axum::response::IntoResponse;
+use tracing::error;
+
+#[derive(Debug)]
+pub struct Error {
```

Comment:
Don't worry _too_ much about this file, just some boilerplate to allow us to do some better errors on the Web side